frontend/src/components/Footer.tsx

```diff
@@ -73,8 +74,10 @@ const Footer: FC<FooterProps> = ({
   menuLinks,
   links,
 }: FooterProps) => {
+  const { pathname } = useRouter()
+  const isTasksPage = pathname.includes('/tasks');
```

Comment:
This should be handle in `Layout` component.

Add a `hideHeader` and `hideFooter` props to Layout component with value of `undefined | "never" | "always" | "print"`. Then you implemtent the logic in `Layout` component then the page should pass the props.